src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -742,7 +749,7 @@ export default function createPostGraphileHttpRequestHandler(options) {
 
       res.setHeader('Content-Type', 'application/json; charset=utf-8')
 
-      res.end(JSON.stringify(returnArray ? results : results[0]))
+      res.end(JSON.stringify(returnArray ? results : results![0]))
```

Comment:
this is a new TS feature i learned recently.  assert that <preceding_var> is defined!

Reply:
This is nice! This has been annoying me for a while 👍